test/unit/mdc-top-app-bar/mdc-top-app-bar.test.js

```diff
@@ -57,16 +57,41 @@ function setupTest() {
   const fixture = getFixture();
   const root = fixture.querySelector('.mdc-top-app-bar');
   const adjust = fixture.querySelector('.mdc-top-app-bar-fixed-adjust');
-  const component = new MDCTopAppBar(root);
-  return {root, adjust, component};
+  const icon = root.querySelector('.mdc-top-app-bar__menu-icon');
+  const component = new MDCTopAppBar(root, undefined, (el) => new FakeRipple(el));
+
+  return {root, adjust, component, icon};
+}
+
+class FakeRipple {
```

Comment:
Can we move this above setupTest? It's jarring to see things defined after they're used.